src/command/subgraph/push.rs

```diff
@@ -86,30 +86,28 @@ impl Push {
 
 fn handle_response(response: PushPartialSchemaResponse, subgraph: &str, graph: &str) {
     if response.service_was_created {
-        tracing::info!(
+        eprintln!(
             "A new subgraph called '{}' for the '{}' graph was created",
-            subgraph,
-            graph
+            subgraph, graph
         );
     } else {
-        tracing::info!(
+        eprintln!(
             "The '{}' subgraph for the '{}' graph was updated",
-            subgraph,
-            graph
+            subgraph, graph
         );
     }
 
     if response.did_update_gateway {
-        tracing::info!("The gateway for the '{}' graph was updated with a new schema, composed from the updated '{}' subgraph", graph, subgraph);
+        eprintln!("The gateway for the '{}' graph was updated with a new schema, composed from the updated '{}' subgraph", graph, subgraph);
     } else {
-        tracing::info!(
+        eprintln!(
             "The gateway for the '{}' graph was NOT updated with a new schema",
             graph
         );
     }
 
     if let Some(errors) = response.composition_errors {
-        tracing::error!(
```

Comment:
this will be printed slightly differently now, i'm wondering if we should have some macros defined in `timber` that would let you do something like:

`timber::info!` `timber::error!` and `timber::warn!` etc. though macro syntax super sucks to work with in rust...

maybe for now I should just add in an `ERROR:` prefix manually?

Reply:
Hmm yea I think for now it would be okay to not implement our own macros. Maybe in the future we can see if that will be a good quality of life investment. I'll be happy with something like `timber.info()`/`timber.error()`/`imber.warn()` for now, and even more immediately with a manual `ERROR:` prefix.